src/bot.ts

```diff
@@ -10,8 +10,8 @@ import { MatrixEventProcessor, MatrixEventProcessorOpts } from "./matrixeventpro
 import { PresenceHandler } from "./presencehandler";
 import * as Discord from "discord.js";
 import * as Bluebird from "bluebird";
-import * as mime from "mime";
 import { Provisioner } from "./provisioner";
+import {DMHandler} from "./dmhandler";
```

Comment:
shouldn't we add spaces on imports to keep whitespace consistent?
```suggestion
import { DMHandler } from "./dmhandler";
```